tests/testthat/test-delineate.R

```diff
@@ -82,8 +82,6 @@ test_that("Only one city and one river can be delineated at a time", {
                "Assertion on 'river_name' failed: Must have length 1")
 })
 
-#' @srrstats {SP6.2} This test checks that delineation is successfully carried
-#'   out for a river-crossed town close to the North Pole.
```

Comment:
If we drop this annotation because the case is not extreme enough, does it still make sense to keep this test?

Reply:
OK, then let's remove it!